lib/mobility/plugins/upload_for_translation.rb

```diff
@@ -0,0 +1,34 @@
+require 'mobility'
+
+module Mobility
+  module Plugins
+    module UploadForTranslation
+      class << self
+        def apply(attributes, option)
+          return unless option
+          # include the write instance method so that we override the backend's write method
+          attributes.backend_class.include self
+        end
+      end
+
+      UPLOAD_TRANSLATION_DELAY = 5.minutes
```

Comment:
nit: we can also put this in ENV for easy configuration, with default to 5 minutes

Reply:
good idea, will fix.